infrastructure/cli/cmd/create/create.go

```diff
@@ -0,0 +1,32 @@
+package create
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	provider  string
+	CreateCmd = &cobra.Command{
+		Use:   "create",
+		Short: "Creates an instance of Airy Core",
+		Long:  ``,
+		Run:   create,
+	}
+)
+
+func init() {
+	CreateCmd.Flags().StringVarP(&provider, "provider", "", "", "One of the supported providers (aws|local)")
+	CreateCmd.MarkFlagRequired("provider")
+	viper.BindPFlag("provider", CreateCmd.PersistentFlags().Lookup("provider"))
+}
+
+func create(cmd *cobra.Command, args []string) {
+	fmt.Println("⚙️  Creating core with provider", provider)
+	fmt.Println("🚀 Starting core with default config")
```

Comment:
Not sure about default config. I would recommend: `Starting core with default components`, as we don't actually provide any configuration for the components by default (the values which are in `airy.yaml`), apart from what is on or off.